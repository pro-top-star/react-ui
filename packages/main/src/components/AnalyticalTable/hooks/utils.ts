export const makeTemplateColumns = (columns, resizedColumns) => {
  return columns
    .map((column) => {
      if (resizedColumns[column.id]) {
        return `${resizedColumns[column.id]}px`;
      }
      return `minmax(${column.minWidth}px, ${column.width || '1fr'})`;
    })
    .join(' ');
};
