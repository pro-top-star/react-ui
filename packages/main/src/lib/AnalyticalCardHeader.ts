import { AnalyticalCardHeader } from '../components/AnalyticalCard/header/AnalyticalCardHeader';

export { AnalyticalCardHeader };
