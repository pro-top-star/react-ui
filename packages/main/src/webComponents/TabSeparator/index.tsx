import { withWebComponent } from '@ui5/webcomponents-react/lib/withWebComponent';
import UI5TabSeparator from '@ui5/webcomponents/dist/TabSeparator';
import React, { FC } from 'react';
import { WithWebComponentPropTypes } from '../../internal/withWebComponent';

export interface TabSeparatorPropTypes extends WithWebComponentPropTypes {}

/**
 * <code>import { TabSeparator } from '@ui5/webcomponents-react/lib/TabSeparator';</code>
 */
const TabSeparator: FC<TabSeparatorPropTypes> = withWebComponent<TabSeparatorPropTypes>(UI5TabSeparator);

TabSeparator.displayName = 'TabSeparator';

export { TabSeparator };
