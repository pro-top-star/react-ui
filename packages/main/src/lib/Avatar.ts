import { Avatar } from '../webComponents/Avatar';

export { Avatar };
