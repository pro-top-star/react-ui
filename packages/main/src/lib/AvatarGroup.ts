import { AvatarGroup } from '../webComponents/AvatarGroup';
import type { AvatarGroupPropTypes } from '../webComponents/AvatarGroup';

export { AvatarGroup };
export type { AvatarGroupPropTypes };
