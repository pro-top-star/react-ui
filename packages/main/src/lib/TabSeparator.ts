import { TabSeparator } from '../webComponents/TabSeparator';
import type { TabSeparatorPropTypes } from '../webComponents/TabSeparator';

export { TabSeparator };
export type { TabSeparatorPropTypes };
