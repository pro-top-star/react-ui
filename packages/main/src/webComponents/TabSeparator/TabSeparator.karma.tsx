import { mountThemedComponent } from '@shared/tests/utils';
import { expect, use } from 'chai';
import { matchSnapshot } from 'chai-karma-snapshot';
import React from 'react';
import { TabSeparator } from '../../lib/TabSeparator';

use(matchSnapshot);

describe('TabSeparator', () => {
  it('Basic Test (generated)', () => {
    const wrapper = mountThemedComponent(<TabSeparator />);
    expect(wrapper.debug()).to.matchSnapshot();
  });
});
