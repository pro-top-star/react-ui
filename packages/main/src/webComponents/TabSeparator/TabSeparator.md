```jsx
import { TabSeparator } from '@ui5/webcomponents-react/lib/TabSeparator';
```
