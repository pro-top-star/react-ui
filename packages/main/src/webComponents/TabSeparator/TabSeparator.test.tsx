import { mountThemedComponent } from '@shared/tests/utils';
import { TabSeparator } from '@ui5/webcomponents-react/lib/TabSeparator';
import React from 'react';

describe('TabSeparator', () => {
  test('Basic Test (generated)', () => {
    const wrapper = mountThemedComponent(<TabSeparator />);
    expect(wrapper.render()).toMatchSnapshot();
  });
});
