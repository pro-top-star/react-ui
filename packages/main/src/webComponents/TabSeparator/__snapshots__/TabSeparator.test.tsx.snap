// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabSeparator Basic Test (generated) 1`] = `<ui5-tab-separator />`;
